src/Libraries/CoreNodeModels/DropDown.cs

```diff
@@ -63,6 +63,22 @@ public ObservableCollection<DynamoDropDownItem> Items
             }
         }
 
+        public override NodeInputData InputData()
+        {
+
+            return new NodeInputData()
+            {
+                Id = this.GUID.ToString("N"),
+                Name = this.Name,
+
+                Type = NodeInputTypes.selectionInput,
+                Description = this.Description,
+                //TODO investigate if this node type also needs to serialize it's selected index.
+                Value = this.Items[this.selectedIndex].Name,
+                Choices = this.items.Select(x => x.Name).ToList(),
```

Comment:
Would be curious about the impact on Graph size serialized in D4R since some of the dropdowns there have a tons of items.